src/services/kuma-api/KumaApi.ts

```diff
@@ -29,13 +32,20 @@ import type {
 import type { ClientConfigInterface } from '@/store/modules/config/config.types'
 
 export default class KumaApi extends Api {
+  setupMockWorker() {
+    if (this._env.var('KUMA_MOCK_API_ENABLED') === 'true') {
+      const handlers = setupHandlers(this.baseUrl, get(TOKENS.mocks))
+      setupMockWorker('KumaApi', handlers)
+    }
+  }
```

Comment:
**Comment**: Responsibilities of the individual `Api` instances are pretty much what they were before (then in encapsulated in the `MockKumaApi` class): setting up API instance-specific mock handlers. This remains necessary as the handlers for various API instances may be constructed differently.